services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequestTest.java

```diff
@@ -69,7 +74,7 @@ public void builder_maximal() {
                                                                .limit(3)
                                                                .queryConditional(queryConditional)
                                                                .scanIndexForward(true)
-                                                               .projectionExpression(projectionExpression)
+                                                               .attributesToProject(attributesToProjectArray)
```

Comment:
When we have the overloaded versions of this, hopefully in the implementation one calls the other, in which case we should make sure we are testing the one that will give us coverage over both. Usually that's the varargs form because that should call the other one with Array.asList(...). We will also need to add a separate test for the .addAttributeToProject(...) method that we are hopefully adding.

Reply:
This is already testing the varargs variant. I will add the one for the `add...` method.